static/app/views/organizationGroupDetails/groupSimilarIssues/similarTraceID/list.tsx

```diff
@@ -31,6 +31,8 @@ type Period = {
   end: string;
 };
 
+type CursorHandler = React.ComponentProps<typeof Pagination>['onCursor'];
```

Comment:
Should CursorHandler be a type that is exported from the pagination component module?

Reply:
Yeah I considered it, 🤔

I'm indifferent. I prefer less exports typically tbh. What do you think?

Reply:
I totally agree with Mark..BUT... we don't usually do that, right?

Reply:
We don't often do it but we have 7 places now that generate the same type. Having an export would save 7 types.

Reply:
Yeah, I'll export, that number convinced me :)

Reply:
updated.